src/components/ChatList/elements/ChannelContent.style.ts

```diff
@@ -9,22 +9,24 @@ export const channelContentStyles = StyleSheet.create({
     fontSize: normalize(14),
     lineHeight: 22,
     flex: 1,
-    paddingBottom: 2
+    paddingBottom: 2,
+    color: COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text has been changed from default to `COLORS.white2`. Ensure that this color change is consistent with the overall theme and accessibility standards. High contrast between text and background colors is important for readability.

```diff
-    paddingBottom: 2
+    paddingBottom: 2,
+    color: COLORS.white2
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->